.travis.yml

```diff
@@ -1,13 +1,11 @@
-language: perl6
-perl6:
-  - latest
-before_install:
-  - if [ "$TRAVIS_PULL_REQUEST" = "false" ]; then echo "Please disable builds on branch updates in settings."; travis_terminate 0; fi
+language:
+  - minimal
+
+services:
+  - docker
 
 install:
-  - rakudobrew build zef
+  - docker pull jjmerelo/perl6-test-ecosystem
+  - docker images
```

Comment:
Why do we need this? Logging?

Reply:
You mean docker images? It's not really needed, it just shows that the image has been downloaded and has installed correctly. You do need to download the image that performs the test, however.